cmd/dockerd/daemon_windows.go

```diff
@@ -93,7 +94,30 @@ func newCgroupParent(config *config.Config) string {
 	return ""
 }
 
-func (cli *DaemonCli) initContainerD(_ context.Context) (func(time.Duration) error, error) {
+func (cli *DaemonCli) initContainerD(ctx context.Context) (func(time.Duration) error, error) {
+	if cli.Config.ContainerdAddr == "" && cli.Config.DefaultRuntime == config.WindowsV2RuntimeName {
+		cli.Config.ContainerdAddr = supervisor.GrpcPipeName
+		system.InitContainerdRuntime(cli.Config.ContainerdAddr)
+
+		var waitForShutdown func(time.Duration) error
+		logrus.Debug("Starting daemon managed containerd")
```

Comment:
In https://github.com/moby/moby/pull/43945, I changed this to

- `managed containerd` (removed "daemon"), and made the logs all lowercase
- changed `Debug` to `Info` (for consistency with some other things we start when starting the daemon)

Perhaps something like;

```go
logrus.Info("starting managed containerd")
```

Or we could even use the same log;

```go
logrus.Info("containerd not running, starting managed containerd")
```